tests/src/logical_comparisons/cmpeq.cpp

```diff
@@ -50,6 +50,7 @@ TEST_CASE("Compare equal - mipp::reg", "[mipp::cmpeq]")
 #endif
 #endif
 	SECTION("datatype = int32_t") { test_reg_cmpeq<int32_t>(); }
+        SECTION("datatype = uint32_t") { test_reg_cmpeq<uint32_t>(); }
```

Comment:
Tabulations are used in MIPP, could you use tabs too :-)? This remark is valid for the whole PR.

Reply:
Oh i recognized that to late. My CLion had the wrong configuration. I'll fix it asap.